### ISSUE TITLE
Client Code & UDP multicast

### ISSUE THREAD

**Issue by Baltor12** [danmilon/node-dpws#1]
Hello,

Could you also please add a sample client code?

Does the server support multicast based discovery?

Thanks in advance.

**Comment by danmilon** [comment]
Yeah, I'll try to dig up something. FYI, I wrote this very hastily for my thesis, and I'm not particularly proud of the code. (like all the XML strings concatenation).

RE: discovery, IIRC it does multicast discovery using [ws-discovery](https://github.com/danmilon/node-ws-discovery).

**Comment by Baltor12** [comment]
Hello,

Thanks for the response. The code looks good. Actually I am doing this as part of my special assignment. I might be doing some changes to your client.js locally. If everything goes well. I will try to contribute to this module.

Thanks.
